circle.yml

```diff
@@ -149,6 +158,8 @@ workflows:
   version: 2
   build-test-and-deploy:
     jobs:
+      - cleanup:
```

Comment:
Remove this before merging. We already have a schedule cleanup tasks below.

Reply:
Sure, I was just keeping this until the approval to make sure none of the changes broke the cleanup job and there's no flakiness. It's quite difficult to test otherwise.